### DISCUSSION TITLE
Issue with importing database in NewPipe

### DISCUSSION THREAD

**Discussion by Omarayoub888** [TeamNewPipe/NewPipe#11769] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

When attempting to import the database (newpipe.db), the app does not detect the file despite placing it in the correct directory. I have granted the necessary storage permissions, and I can see the database files in the appropriate folder. However, I am unable to click on or select the newpipe.db file to import it. The import option remains unavailable even though the file is visible.

### Additional information

I have a Samsung Galaxy M51, and before formatting my phone, I exported the NewPipe database to restore it afterward, but I am unable to import it after reinstallation.

**Comment by asandikci** [comment, marked as answer]
What was the app version when you got the export? And what is the current version?

And could you open it with BendyStraw app? Maybe it could help

**Reply by Omarayoub888** [reply]
Thank you so much for your suggestion! Using BendyStraw worked perfectly, and I was able to recover my data. I really appreciate your help!
